call_test.go

```diff
@@ -225,7 +225,7 @@ func TestInvokeLargeErr(t *testing.T) {
 	var reply string
 	req := "hello"
 	err := Invoke(context.Background(), "/foo/bar", &req, &reply, cc)
-	if _, ok := err.(rpcError); !ok {
+	if _, ok := err.(*rpcError); !ok {
```

Comment:
what happens though if we get a `rpcError` or an `*rpcError`, only the `*rpcError` will be accepted. IMO we can keep compatibility by defining a small function like this
```go
func rpcErrorLike(e interface{}) bool {
  switch e.(type) {
  case *rpcError, rpcError:
     return true
  default:
     return false
  }
}
```

and then use it in those checks this way the code will be compatible with all users
so we'll change it to
```go
if !rpcErrorLike(err) {
   t.Fatalf("grpc.Invoke(_, _, _, _, _) receives non rpc error.")
}
```

Let me know what you think.